lib/board.rb

```diff
@@ -0,0 +1,83 @@
+require_relative './game.rb'
+
+class Board < Game
+  def initialize
+    @game = Game.new(@players)
+  end
+
+  def welcome_message
+    puts 'Welcome to Tic Tac Toe ! ! !'
+  end
+
+  def user_setup
+    print 'Player 1: What is your name?:'
+    @player_x = gets.chomp
+    @player_x.upcase!
+    name = true
+    while name
+      print 'Player 2: What is your name?:'
+      @player_y = gets.chomp
+      @player_y.upcase!
+      if @player_y != @player_x
+        name = false
+      else
+        puts 'Please Choose Different Name'
+      end
+    end
+  end
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- the content of this method should be completely put in the main.rb file or it should be refactored so it doesn't use puts, prints, and these are placed in the main.rb file